help/moving-to-zulip.md

```diff
@@ -0,0 +1,207 @@
+# Moving to Zulip
+
+Welcome to Zulip! This page will guide you through the process of transitioning
+your organization to Zulip. It assumes that you have [completed you initial
+evaluation](/help/trying-out-zulip) of Zulip, decided [whether to use Zulip
+Cloud or self-host](/help/zulip-cloud-or-self-hosting), and are ready to
+introduce Zulip to your organization.
+
+The following steps are described in more detail below:
+
+{start_tabs}
+
+1. [Create your organization](#create-your-organization).
+
+1. [Sign up for a plan](#sign-up-for-a-plan).
+
+1. [Configure your organization](#configure-your-organization).
+
+1. [Prepare users for the transition](#prepare-users-for-the-transition).
+
+1. [Invite users to join](#invite-users-to-join).
+
+1. [Update your guidelines](#update-your-guidelines).
+
+{end_tabs}
+
+Each organization is unique, but we hope these common practices will help you
+think through the transition process in your own context.
+
+## Create your organization
+
+You can create a new Zulip Cloud organization in less two minutes. Setting up a
+self-hosted server will take a bit longer, but is easy to do with Zulip's
+[well-documented](https://zulip.readthedocs.io/en/stable/production/install.html)
+scripts.
+
+Zulip has import tools for [Slack](/help/import-from-slack),
+[Mattermost](/help/import-from-mattermost) and
+[Rocket.Chat](/help/import-from-rocketchat). You can import your organization's
+chat data, including message history, users, channels, and custom emoji. To
+inquire about importing data from another product, [contact Zulip
+support](/help/contact-support).
+
+Data is imported into Zulip as a new organization, so the best time to import is
+when your team is about to start using Zulip for day-to-day work. This may be
+part of your evaluation process, or after you've made the decision to move to
+Zulip.
+
+{start_tabs}
+
+{tab|new-organizations}
+
+1. If you plan to self-host, [set up your Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html). You
+   can self-host Zulip directly on Ubuntu or Debian Linux, in
+   [Docker](https://github.com/zulip/docker-zulip), or with prebuilt images for
+   [Digital Ocean](https://marketplace.digitalocean.com/apps/zulip) and
+   [Render](https://render.com/docs/deploy-zulip).
```

Comment:
This block should probably be done with the include mechanism since it seems identical.

Reply:
I think it's not worth doing for 2 blocks within the same page.